middleware/localNamingConventions.js

```diff
@@ -0,0 +1,39 @@
+
+function setup() {
+  return applyLocalNamingConventions;
+}
+
+function applyLocalNamingConventions(req, res, next) {
+
+  // do nothing if no result data set
+  if (!res || !res.data) {
+    return next();
+  }
+
+  // loop through data items and flip relevant number/street
+  res.data.filter(function(place){
+    // only relevant for German addresses
+    if( 'DEU' !== place.alpha3 ){ return false; }
```

Comment:
I think this should be more config-driven. `DEU` and filter names should be part of a config file where you specify which get which analyzers. For example:

```javascript
{
  "DEU": ["flipNumberAndStreet"]
}
```

then in code something like:

```javascript
if (check.isSet(config, place.alpha3)) {
  filters[config[place.alpha3](place);
}
```